packages/sanity/src/core/studio/tasks/context/tasks/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface TasksContextValue {
```

Comment:
I think it would be good if these were a bit more UI agnostic. Even though the current implementation is a sidebar, I believe keeping it more generic with `open` and `setOpen` would be good, as we might want to show the tasks in, for example, a dialog in the future. 

```ts

const {open, setOpen} = useTasks()

```